korge/src/commonMain/kotlin/com/soywiz/korge/tween/tween.kt

```diff
@@ -109,8 +109,8 @@ data class V2<V>(
 	internal val initial: V,
 	internal val end: V,
 	internal val interpolator: (V, V, Double) -> V,
-	internal val startTime: Int = 0,
-	internal val duration: Int? = null
+	internal val startTime: Long = 0,
+	internal val duration: Long? = null
```

Comment:
Ditto double boxing on JS. We could consider something like using Doubles and `TimeSpan` and a constant like `val NO_DURATION = -1.0`